03 Algorithm Reference/15 Reality Modelling/04 Fee Models.html

```diff
@@ -50,7 +50,7 @@
 </pre>
 <pre class="python" >
 # Assigning securities custom fee models:
-self.Securities["SPY"].SetFeeModel(CustomFeeModel())
+self.Securities["SPY"].SetFeeModel(CustomFeeModel)
```

Comment:
This one actually needs the **"instance"** of the custom fee model set; not the static class reference. So you need the braces()

I doubt this would pass a test in the IDE; can you please make sure to test all PR's submitted.

Reply:
Earlier I had this error: https://quantconnect.slack.com/archives/GF6ERSYBG/p1594145344224800?thread_ts=1594145204.224600&cid=GF6ERSYBG, but retested with essentially the same code, and error is gone